README.rst

```diff
@@ -192,19 +192,17 @@ MFE development
 
 Tutor makes it possible to run any MFE in development mode. For instance, to run the "profile" MFE::
 
-    tutor dev runserver profile
+    tutor dev start profile
 
 Then, access http://apps.local.overhang.io:1995/profile/u/YOURUSERNAME
 
-To run your own fork of an MFE, start by copying the MFE repo on the host::
+You can also bind-mount in your own fork of an MFE. For example::
 
-    tutor dev bindmount profile /openedx/app
+    cd /path/to/frontend-app-profile
+    npm install  # Ensure NPM requirements are installed into your fork.
+    tutor dev start profile --mount .
```

Comment:
I believe this line should be:

    tutor dev start --mount=. profile

Reply:
Click allows either syntax, but I'll change it just to be consistent with the Tutor's dev docs on mounting.

Reply:
Done